### ISSUE TITLE
The new `run-rerun` command is throwing an error when running BDD files

### ISSUE THREAD

**Issue by GSasu** [Codeception/CodeceptJS#2133]
#### What are you trying to achieve?
I am trying to use the new `run-rerun` command to run one or multiple BDD scenarios.

#### What do you get instead?
```
internal/modules/cjs/loader.js:783
    throw err;
    ^

Error: Cannot find module 'arrify'
Require stack:
- /Users/gabe/Development/service-automation-framework/node_modules/parse-function/dist/cjs/index.js
- /Users/gabe/Development/service-automation-framework/node_modules/codeceptjs/lib/parser.js
- /Users/gabe/Development/service-automation-framework/node_modules/codeceptjs/lib/command/list.js
- /Users/gabe/Development/service-automation-framework/node_modules/codeceptjs/bin/codecept.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:780:15)
    at Function.Module._load (internal/modules/cjs/loader.js:685:27)
    at Module.require (internal/modules/cjs/loader.js:838:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/Users/gabe/Development/service-automation-framework/node_modules/parse-function/dist/cjs/index.js:5:34)
    at Module._compile (internal/modules/cjs/loader.js:945:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:962:10)
    at Module.load (internal/modules/cjs/loader.js:798:32)
    at Function.Module._load (internal/modules/cjs/loader.js:711:12)
    at Module.require (internal/modules/cjs/loader.js:838:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/gabe/Development/service-automation-framework/node_modules/parse-function/dist/cjs/index.js',
    '/Users/gabe/Development/service-automation-framework/node_modules/codeceptjs/lib/parser.js',
    '/Users/gabe/Development/service-automation-framework/node_modules/codeceptjs/lib/command/list.js',
    '/Users/gabe/Development/service-automation-framework/node_modules/codeceptjs/bin/codecept.js'
  ]
}
```
### Details

* CodeceptJS version: 2.4
* NodeJS Version: 12.11
* Operating System: Mac Os 10.15.2
* webdriverio: ^5.18.5

**Comment by PeterNgTr** [comment]
This issue may be solved when this [PR](https://github.com/Codeception/CodeceptJS/pull/2134) is live

**Comment by GSasu** [comment]
I can confirm that this issue i fixed by installing "arrify" with `npm i --save arrify` thanks to @thanh on Slack

**Comment by elukoyanov** [comment]
Fixed in release 2.4.1
